.github/workflows/flakeguard.yml

```diff
@@ -329,7 +329,7 @@ jobs:
         shell: bash
         run: |
           go install github.com/smartcontractkit/chainlink-testing-framework/tools/flakeguard@9687993689805382c40dd48250a07efd155a09b9 # june 11, 2025
-          go install gotest.tools/gotestsum@v1.12.2 # needed for flakeguard output formatting
+          go install gotest.tools/gotestsum@v1.12.3 # needed for flakeguard output formatting
```

Comment:
Nit: Can remove flakeguard comment

Reply:
just kidding

Reply:
technically this workflow is still using flakeguard.